addons/product/models/product_attribute.py

```diff
@@ -17,7 +17,7 @@ class ProductAttribute(models.Model):
 
     name = fields.Char('Attribute', required=True, translate=True)
     value_ids = fields.One2many('product.attribute.value', 'attribute_id', 'Values', copy=True)
-    sequence = fields.Integer('Sequence', help="Determine the display order", index=True)
+    sequence = fields.Integer('Sequence', help="Determine the display order", index=True, default=20)
```

Comment:
```suggestion
    sequence = fields.Integer('Sequence', help="Determine the display order", index=True, default=10)
```
Doesn't matter much, but it's the most used value, and I prefer if we use that value unless there is a specific need for a different number.

Reply:
I tried 10 but it broke tests that expect a specific order of certain attributes because of the sequence in some of the demo data.
Since it doesn't matter much it made more sense to me to change than "fix" the tests

Reply:
Oh, ok then, if there is a reason for it :D